src/osx/carbon/colordlgosx.mm

```diff
@@ -55,7 +55,7 @@ @implementation wxCPWCDelegate : NSObject
 
 - (id)init
 {
-    if ( self = [super init] )
+    if ((self = [super init]))
```

Comment:
IMHO this looks bad/unidiomatic. If we can't suppress the warning, I'd rather write this as

```suggestion
    self = [super init];
    if (self)
```